src/map_server/include/map_server/base_map_loader.h

```diff
@@ -0,0 +1,37 @@
+#ifndef MAP_SERVER_BASE_MAP_LOADER_H
```

Comment:
You'll have to make some changes to accomodate the new directory structure. For example, map_server becomes nav2_map_server, affecting include directories, header macro guards, etc.